package.json

```diff
@@ -1,52 +1,67 @@
 {
-  "name": "tigris-client-ts",
-  "version": "1.0.0-alpha.1",
-  "description": "Tigris client for Typescript",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/tigrisdata/tigris-client-ts"
-  },
-  "scripts": {
-    "clean": "rm -rf ./src/proto/* && rm -rf dist && rm -rf node_modules",
-    "update_api": "git submodule update --init --recursive && git submodule update --remote --recursive --rebase && git submodule foreach --recursive git reset --hard origin/main",
-    "protoc": "./scripts/protoc.sh",
-    "lint": "node ./node_modules/eslint/bin/eslint src/**/*.ts",
-    "lint-fix": "node ./node_modules/eslint/bin/eslint --fix src/**/*.ts",
-    "copy_api": "copyfiles -u 2 \"./src/proto/**/*.{js,ts}\" \"./dist/proto\"",
-    "tsc": "tsc && npm run copy_api",
-    "build": "npm install && npm run update_api && npm run protoc && npm run tsc",
-    "test": "jest --runInBand",
-    "all": "npm run clean && npm run update_api && npm run protoc && npm run lint && npm run build"
-  },
-  "engines": {
-    "node": ">= 12.0.0"
-  },
-  "keywords": [
-    "typescript",
-    "jest",
-    "mock"
-  ],
-  "devDependencies": {
-    "@types/jest": "^27.4.1",
-    "@typescript-eslint/eslint-plugin": "^5.21.0",
-    "@typescript-eslint/parser": "^5.23.0",
-    "copyfiles": "^2.4.1",
-    "eslint": "^8.15.0",
-    "eslint-plugin-functional": "^4.2.1",
-    "eslint-plugin-unicorn": "^42.0.0",
-    "grpc_tools_node_protoc_ts": "^5.3.2",
-    "grpc-tools": "^1.11.2",
-    "jest": "^27.0.4",
-    "ts-jest": "^27.0.3",
-    "tsutils": "^3.21.0",
-    "typescript": "^4.7.2",
-    "uuid": "^8.3.2"
-  },
-  "dependencies": {
-    "@grpc/grpc-js": "^1.6.7",
-    "google-protobuf": "^3.20.1",
-    "json-bigint": "^1.0.0"
-  }
+	"name": "@tigrisdata/core",
+	"version": "1.0.0-alpha.1",
+	"description": "Tigris client for Typescript",
+	"author": "Tigris Data (https://www.tigrisdata.com/)",
+	"contributors": [
+		{
+			"name": "A Ibrahim",
+			"email": "hello@designcode.me",
+			"url": "http://designcode.me"
+		}
+	],
```

Comment:
It makes code review really hard when the entire file appears to be modified even the only some parts of it is modified. please keep it mind for next time.

Reply:
Indeed, I've added editor.config and it seemed to have auto formatted whole file.